test/run/import-module-explicit.mo

```diff
@@ -1,5 +1,5 @@
-import { cons; nil } = "lib/ListM";
+import { cons; nil = empty } = "lib/ListM";
```

Comment:
@crusso indeed, this works! 🥳

Reply:
Yeah, that's nice. But what if they want to refer to the List.List type? 

Absent type fields, we could solve that with simple alias " <pat> as <id>`patterns (a la OCaml) or more general `<pat> and <pat>` patterns, I guess. That can come later.

Reply:
The will need a second `import` for now. Is that posssible? I guess so.

Reply:
Like this:
``` Motoko
import { cons; nil = empty } = "lib/ListM";
import L = "lib/ListM";

type stack = L.List<Int>;
```
and this works.